.github/workflows/ci.yml

```diff
@@ -470,7 +470,7 @@ jobs:
 
               # check pure pandas
               MODIN_ASV_USE_IMPL=pandas asv run --quick --strict --show-stderr --launch-method=spawn \
-                -b ^benchmarks -b ^io -b ^scalability | tee benchmarks.log
```

Comment:
These benchmarks are supposed to be run only for modin, since they do not make sense for pandas.